configure.ac

```diff
@@ -1,8 +1,8 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.66])
-AC_INIT(swupd-client, 3.19.0, otavio.pontes@intel.com)
+AC_PREREQ([2.69])
```

Comment:
why did you change that? (I'm more curious than concerned)

Reply:
I don't think I did.... maybe a rebase did it?